clients/mobile/__tests__/authentication/authService.test.ts

```diff
@@ -234,6 +238,90 @@ describe('AuthService', () => {
     })
   })
 
+  describe('resetPasswordHandler', () => {
+    it('should return nextStep on success', async() => {
+      const mockNextStep = { step: 'CONFIRM_RESET' }
+      ;(resetPassword as jest.Mock).mockResolvedValueOnce({ nextStep: mockNextStep })
+
+      const result = await resetPasswordHandler(mockEmail)
+      expect(result).toEqual(mockNextStep)
+      expect(resetPassword).toHaveBeenCalledWith({ username: mockEmail })
+    })
+
+    it('should throw an error with a custom message on failure', async() => {
+      const errorMessage = 'Network error'
+      ;(resetPassword as jest.Mock).mockRejectedValueOnce(new Error(errorMessage))
+
+      await expect(resetPasswordHandler(mockEmail)).rejects.toThrow(errorMessage)
+      expect(resetPassword).toHaveBeenCalledWith({ username: mockEmail })
+    })
+  })
+
+  describe('handleConfirmPasswordError', () => {
+    const handleConfirmPasswordError = (error: unknown): string => {
+      if (error instanceof Error) {
+        switch (true) {
```

Comment:
no reason for switch if you're using a boolean

Reply:
Removed switch  [Commit](https://github.com/CraftsmenLtd/BloodConnect/pull/202/commits/ba27593e9bda7d902822fc85881238bf5b32500a)